wazo_websocketd/tests/test_bus.py

```diff
@@ -213,36 +222,35 @@ def setUp(self):
         self.origin_uuid = Mock()
         self.mock_config = dict(_DEFAULT_CONFIG, uuid=self.origin_uuid)
 
-    def _make_consumer(self, purpose: str, admin: bool | None = None):
-        mock_token: dict[str, Any] = {
-            'session_uuid': Mock(),
-            'metadata': {
-                'uuid': 'some-user-uuid',
-                'tenant_uuid': Mock(),
-                'purpose': purpose,
-            },
-            'acl': ['some.acl'],
-        }
-
+    def _make_consumer(
+        self,
+        purpose: str,
+        admin: bool | None = None,
+        user_uuid: str | None = None,
+    ):
+        mock_token: TokenDict = _DEFAULT_TOKEN
+        mock_token['metadata']['uuid'] = user_uuid
+        mock_token['metadata']['purpose'] = purpose
```

Comment:
Won't this modify the global dict though? Perhaps the default token could be in a function so that it is always a different instance? Or even mark it as a fixture:

```python
@pytest.fixture
def mock_token() -> TokenDict:
    return {... # the token}
```

Then you could easily include it if using functions, or globally for the class with the `@pytest.mark.usefixtures` decorator we discussed the other day.

Reply:
Oh wow, such bad error, sorry :disappointed: 
Quickly tried to use fixture, but would require more refactor than I want to do for typing
But I agree with you, so feel free to make the change if you really want it

Reply:
Haha, no worries. That's what reviews are for.
Ok, I added [a commit](https://github.com/wazo-platform/wazo-websocketd/pull/80/commits/bdb04a64009c39c55119a8734a95e2dd278739c1) for that if you want to have a look. We can revert it if you don't like it. Fixtures are extremely limited when used with classes sadly, which is why using functions is preferred.

Reply:
Oh nice one.
I didn't know that we could use fixture like this.
LGTM, you can merge it

Reply:
Ok, cool. I'll approve it then :) Yeah, it's always a bit of a hack with classes though, which is why I'm not a huge fan of TestCase.